.pre-commit-config.yaml

```diff
@@ -1,11 +1,11 @@
 ---
 repos:
-  - repo: git://github.com/Lucas-C/pre-commit-hooks
```

Comment:
I would split these changes into a separate commit (because they're not really related to the other changes here and deserve their own description).